allure-pytest/examples/step/step.rst

```diff
@@ -10,6 +10,11 @@ Steps
     ...         pass
 
 
+    >>> def test_inline_fail_mark_step():
+    ...     with allure.step("inline fail mark step") as step:
+    ...         step.fail_mark = "some fail message"
```

Comment:
I suggest you to change API in order to split message & status properties as you may want to add message to passed steps, or use broken status instead of failed:

```
step.status = Status.FAILED
step.message = "some message"
step.trace = "some trace"
```

Reply:
Hi, thanks for feedback. I agreed and push changes

Reply:
Hi Dmitry @baev , maybe you have any else questions or suggestions?

Reply:
nope, all good